test/integration_test_helper.rb

```diff
@@ -31,6 +31,7 @@
 Capybara.configure do |config|
   config.javascript_driver      = ENV["JS_TEST_DRIVER"]&.to_sym || :selenium_chrome
   config.default_max_wait_time  = 20
+  config.enable_aria_label = true
```

Comment:
@ezr-ondrej, can I just add things to ruby tests of do I need a new pr/discussion? :)
> **enable_aria_label** (Boolean = `false`) - Whether fields, links, and buttons will match against `aria-label` attribute.

Reply:
I'd like this to be a separate commit, but just to appease my OCD. If it doesn't change or affect behaviour of other tests radically, you're fine to add simple config changes like this :+1: